### PR TITLE
Hello microverse project

### PR DESCRIPTION
**Pull request by nedith**
Changes made:-
-Added node_modules directory to .gitignore file
-Removed empty space on line 1 in style.css file
-Added 2 spaces on lines 2 and 3 in style.css file
-Added a new empty line on line 5 in style.css file
-Removed empty lines at the end of the README file

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+test.md
+linters.yml
+
```
Comment (OLIPLICHE) [inline .gitignore]: - [ ] Please add `node_modules` inside `.gitignore` file

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @nedith , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- README file is descriptive 👏
Commit messages are meaningful 👏 👏
Github flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏

## Required Changes ♻️

- [ ] Please make the Linter result should be green (the last commit have a green check mark (✅))

- [ ] Please make sure the project Pull Request have a summary describing introduced changes in more detail. Open this link to learn more. ([Basic rule to follow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md))

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Remove the unsused section in your Readme to make it looks cleaner.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by nedith** [comment]
Fixed issues in style.css:
-Removed empty space on line 1
-Added 2 spaces on lines 2 and 3
-Added a new empty line on line 5

File: README.md [hunk 2]
```diff
@@ -1 +1,40 @@
-# hello-microverse
\ No newline at end of file
+# Hello-Microverse
```
Comment (Ismailco) [inline README.md:1]: - [ ] [Optional] This PR has no description, as the previous reviewer mentioned please make sure the project Pull Request has a summary describing introduced changes in more detail. Open this link to learn more. ([Basic rule to follow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)).

Screenshot_20220524_175621

- ####  You can add a description by clicking on the ```...``` at the top right of the description box, then choosing ```edit``` button
Screenshot_20220524_175736

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @nedith,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @nedith 

Your project is complete! Great job for Adding the changes as requested by the previous reviewer 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- Validations correctly implemented
- No linter errors

<be>

Optional suggestions

- [optional ] please try to make your pull request description summary to focus on the ideas and tasks added to the project or  main things you accomplished in the project. 😄 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**